src/externs.rs

```diff
@@ -20,6 +20,17 @@ pub fn set_external(ext: Box<External>) {
 	});
 }
 
+pub fn get_external<T: External + Clone + 'static>() -> T {
+	EXTERNAL.with(|arg| {
+		let ref_cell: &RefCell<Box<External>> = arg;
```

Comment:
I'm wondering, is it possible to change `|arg|` to `|ref ref_cell|`?

Reply:
I'll check )